### ISSUE TITLE
Nuget version 0.9.0 seems to be outdated

### ISSUE THREAD

**Issue by ollejacobsen** [JudahGabriel/RavenDB.ModernMvcStarterKit#1]
The version (0.9.0) published on Nuget seem to be outdated. 
Ie the IdentityUser class is missing a bunch of needed properties.

Thanks for the project!

**Comment by JudahGabriel** [comment]
Yeah, I think MS Identity Framework has been updated. I'll update the project with the new properties shortly.
